less-print/viewdetails.less

```diff
@@ -1,5 +1,90 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+  display: none !important;
+}
+
+.view-details-title{
+	display: block;
+	font-size: .75em !important;
+	margin-top: 0px;
+	margin-bottom: 10px;
+}
+
+.description , .view-details-info {
+	display: block !important;
```

Comment:
I've made some changes : 
- Firefox displaying bug is solved (on my tests)
- No more thumbnails
- Small modifications about the h3 balise and images which were split on two pages

It take a lot of time to make tests with different details views (routes, outings, articles) and different contents (short or long description, images embedded or not, headers, lists...).
I think there could be a bug with Chrome : if content takes a line more than the first page, this line may be not displayed.
Maybe the way texts are broken can by improved.

![image](https://cloud.githubusercontent.com/assets/24532066/21235413/eca59630-c2f6-11e6-8d8d-631315db7001.png)
![image](https://cloud.githubusercontent.com/assets/24532066/21235427/fa8f809e-c2f6-11e6-8ee7-7b50c305d945.png)

![image](https://cloud.githubusercontent.com/assets/24532066/21235476/2dff1520-c2f7-11e6-9619-b4483199934d.png)
![image](https://cloud.githubusercontent.com/assets/24532066/21235492/46c764ea-c2f7-11e6-92a7-3ffb56b85775.png)
![image](https://cloud.githubusercontent.com/assets/24532066/21235498/50744c92-c2f7-11e6-9cc6-f998c687deab.png)

Even the home page is better :) (only tested with Chrome). I havn't done anything to improve the printing of the home page, but I wanted to see how it was : 
![image](https://cloud.githubusercontent.com/assets/24532066/21235577/a38320ca-c2f7-11e6-9d22-9885936da9c7.png)